java/src/jmri/jmrix/loconet/LnCommandStationType.java

```diff
@@ -33,40 +33,61 @@
 public enum LnCommandStationType {
 
     //  enum value(name, canRead, progEndOp, ThrottleManager, SlotManager, supportsIdle, supportsMultimeter
-    COMMAND_STATION_DCS100("DCS100 (Chief)", true, false, "LnThrottleManager", "SlotManager", true, false), // NOI18N
-    COMMAND_STATION_DCS240("DCS240 (Advanced Command Station)", true, false, "LnThrottleManager", "SlotManager", true, true), // NOI18N
-    COMMAND_STATION_DCS210("DCS210 (Evolution Command Station)", true, false, "LnThrottleManager", "SlotManager", true, true), // NOI18N
-    COMMAND_STATION_DCS200("DCS200", true, false, "LnThrottleManager", "SlotManager", true, false), // NOI18N
-    COMMAND_STATION_DCS050("DCS50 (Zephyr)", true, false, "LnThrottleManager", "SlotManager", false, false), // NOI18N
-    COMMAND_STATION_DCS051("DCS51 (Zephyr Xtra)", true, false, "LnThrottleManager", "SlotManager", false, false), // NOI18N
-    COMMAND_STATION_DCS052("DCS52 (Zephyr Express)", true, false, "LnThrottleManager", "SlotManager", false, false), // NOI18N
-    COMMAND_STATION_DB150("DB150 (Empire Builder)", false, true, "LnThrottleManager", "SlotManager", true, false), // NOI18N
+    COMMAND_STATION_DCS100("DCS100 (Chief)", true, false, "LnThrottleManager", // NOI18N
+            "SlotManager", true, false, false), // NOI18N
```

Comment:
The list will be much easier to read and maintain if constants or enums like CAN_READ and CAN_NOT_READ is used instead of true and false. If enums like CanRead and CanNotRead is used, type safety will also be added which is useful when the number of parameters are increasing.